src/backend/serviceConfig.ts

```diff
@@ -66,7 +66,7 @@ if (process.env.ENV === 'local') {
         enslig_iverksett: `http://familie-ef-iverksett`,
         kontantstøtte_sak: `http://familie-ks-sak`,
         barnehagelister_api: 'http://familie-ks-barnehagelister',
-        tilbake: `http://familie-tilbake`,
+        tilbakekreving: `http://tilbakekreving-backend`,
```

Comment:
Her må vi skille mellom dev og prod i overgangsfasen

Reply:
Håndterer det ved å ha tilbakekreving og familie_tilbake som to separate blokker - hvor de lever i hver sin prosesseringside.